### ISSUE TITLE
Cannot utilize plugin settings

### ISSUE THREAD

**Issue by eddively** [aleen42/gitbook-footer#2]
The plugin settings window that pops up in Gitbook Editor cannot be edited/saved.

If I ignore those settings and simply try to edit the settings using pluginsCongif, the default is overriden with the default aleen42 info and QR code.

**Comment by aleen42** [comment]
Get it! I'll check the bug soon for you!

**Comment by aleen42** [comment]
@eddively , what is the version of your Gitbook editor?

**Comment by aleen42** [comment]
@eddively , If you are using a editor over 5.x version, there is a terrible bug for this editor like the issue [#1315](https://github.com/GitbookIO/gitbook/issues/1315). When you have a large book, it's too slow to save a file, as well as the `book.json`. Therefore, if you click the `edit` button, it should be a long time to override the file `book.json`.

**Comment by aleen42** [comment]
```json
{
    "plugins": [
        "page-footer"
    ],
    "pluginsConfig": {
        "page-footer": {
            "wisdom": "More than a coder, more than a designer",
            "copyright": "Copyright © aleen42",
            "author": "Aleen",
            "qrcode": true,
            "style": "symmetrical",
            "timeColor": "#a10000",
            "utcOffset": "8",
            "format": "YYYY-MM-DD HH:mm:ss",
            "baseUri": "https://aleen42.gitbooks.io/personalwiki/content/",
            "copyThrightColor": "#666",
            "description": "update time: "
        }
    }
}
```

This is my structure of `book.json`

**Comment by aleen42** [comment]
@eddively, Gitbook has used 3.0.3 version to build books recently and this engine has some deprecated methods of reading configs, so you can temporarily use engine 2.6.7 like this configuration (`book.json`), and I'll fix this compatibility problems soon later:

```json
{
    "gitbook": "2.6.7",
    "plugins": [
        "page-footer"
    ],
    "pluginsConfig": {
        "page-footer": {
            "wisdom": "More than a coder, more than a designer",
            "copyright": "Copyright © aleen42",
            "author": "Aleen",
            "qrcode": true,
            "style": "symmetrical",
            "timeColor": "#a10000",
            "utcOffset": "8",
            "format": "YYYY-MM-DD HH:mm:ss",
            "baseUri": "https://aleen42.gitbooks.io/personalwiki/content/",
            "copyThrightColor": "#666",
            "description": "update time: "
        }
    }
}
```

**Comment by aleen42** [comment]
@eddively , page-footer 4.2.9 has fixed the problems about engine Gitbook 3.0.3~ :facepunch:

**Comment by noerw** [comment]
some settings (like `author` and `qrcode`) are not applied. neither with gitbook v2.6.4 or v.3.0.3
